packages/ember/index.ts

```diff
@@ -633,39 +633,40 @@ Object.defineProperty(Ember, 'TEMPLATES', {
   enumerable: false,
 });
 
-const deprecateImportFromString = function (
-  name: string,
-  message = `Importing ${name} from '@ember/string' is deprecated. Please import ${name} from '@ember/template' instead.`
-) {
-  deprecate(message, false, {
-    id: 'ember-string.htmlsafe-ishtmlsafe',
-    for: 'ember-source',
-    since: {
-      available: '4.10',
-      enabled: '4.10',
-    },
-    until: '5.0.0',
-    url: 'https://deprecations.emberjs.com/v3.x/#toc_ember-string-htmlsafe-ishtmlsafe',
-  });
-};
-// NOTE: these are expressly *not* in the public API, because they were
-// deprecated and removed. TODO: remove them after we land the TS conversion,
-// and after confirming doing so is safe -- the state of the `@ember/string`
-// conversion remains confused.
-Object.defineProperty(Ember.String, 'htmlSafe', {
-  enumerable: true,
-  configurable: true,
-  get() {
-    deprecateImportFromString('htmlSafe');
-    return htmlSafe;
-  },
-});
-Object.defineProperty(Ember.String, 'isHTMLSafe', {
+function deprecateStringUseOnEmberModule() {
```

Comment:
The prior deprecation here would show `Importing htmlSafe from '@ember/string' is deprecated. Please import htmlSafe from '@ember/template`, but would only show when someone was doing 

```
import Ember from 'ember';
Ember.String.htmlSafe('foo');
```

Now, they will see 'Using `Ember.String` is deprecated. Please import methods directly from `@ember/string`' If they do that with `htmlSafe` or `isHTMLSafe` they will then get a deprecation about importing from `@ember/template`. It's a little annoying but I think it is the cleanest way forward and most projects will have updated the imports already.